aten/src/ATen/mps/MPSStream.mm

```diff
@@ -11,6 +11,11 @@ @interface MPSGraphExecutionDescriptor ()
 namespace at {
 namespace mps {
 
+#define MB(x) (x * 1048576UL)
```

Comment:
Why do we have to define this again, didn't we have this defined somewhere in MPSAllocator.h

Reply:
Moved it to a common place in MPSAllocatorInterface.h